examples/window/window_settings.rs

```diff
@@ -16,6 +16,7 @@ fn main() {
                 height: 300.,
                 present_mode: PresentMode::AutoVsync,
                 always_on_top: true,
+                cursor_grab_mode: CursorGrabMode::Confined,
```

Comment:
Does the example here need to be changed? Feels like this just adds noise to the example without an explanation why users should set this. If it should, we should document what this is actually doing so that people learning from the example can improve their understanding.

Reply:
I just want to show users they could set cursor grab mode directly. But they will do this without guidance since this field is already in `WindowDescriptor`. I will remove this change. Thanks!